### DISCUSSION TITLE
i3wm on multi monitor

### DISCUSSION THREAD

**Discussion by Coofle420** [i3/i3#5598] [Q&A] [answered]
hello everyone first time posting to the discussion forum but i was wondering if there was a video or documentation telling me on how to use i3 on 2 monitors thank you guys for your help and also i am very new tiling window managers i have used them in the past but they were just scripts built for me so I thought I do it myself

**Comment by orestisfl** [comment, marked as answer]
The userguide has a chapter, take a look and improvements are always welcome if you have any suggestions

https://i3wm.org/docs/userguide.html#multi_monitor
